modules/home-environment.nix

```diff
@@ -309,11 +309,11 @@ in
             relativePath="''${sourcePath#$oldGenFiles/}"
             targetPath="$HOME/$relativePath"
             if [[ -e "$newGenFiles/$relativePath" ]] ; then
-              $VERBOSE_ECHO "Checking $targetPath  exists"
+              $VERBOSE_ECHO "Checking $targetPath exists"
```

Comment:
My intention here was to have some form of separation between the "checking" part and the "result" part. But I agree that two spaces is silly. Would something like `"Checking $targetPath ... exists"` and `"Checking $targetPath ... gone (deleting)"` be more clear?